app/controllers/planned_runs_controller.rb

```diff
@@ -54,9 +57,24 @@ def update
 
     #Update all runs if Planned_run is changed
     Run.where(planned_run_id: @planned_run.id).find_each do |run|
-      run.date = @planned_run.date
-      run.training_plan = @planned_run.training_plan
-      run.save
+      update = true
```

Comment:
Renamed to updateRun: https://github.com/o19s/thats-trackable/commit/418a5cb4c4f36b36e4a9569be9d9b3020e8423ef